communication/contributor-comms/storytelling-resources/README.md

```diff
@@ -0,0 +1,37 @@
+# Blogging resources
+
+We have collected some information to get anyone started. If you're looking to get started blogging on the kubernetes.dev blog, this page contains a list of resources available to help you be successful.
+
+## [Blog guidelines](./blog-guidelines.md)
+
+Information on the type of content, possible publication targets, and
+a description of the editing and publishing process.
+
+## [SIG Spotlight topics](./example-topics.md)
+
+Ready to use topics to use in the SIG Spotlight series.
+
+## [SIG Spotlight example questions](./example-interview-qs-sig-windows.md)
+
+Examples of the questions used in the SIG Windows spotlight.
+
+## [SIG Spotlight blog process](./sig-spotlight-blog-process.md)
+
+A step-by-step description on how to do a SIG Spotlight blog article.
+
+## [SIG Spotlight series: covered SIGs](./sig-spotlights.md)
+
+List of already covered SIGs, with a link to the blog articles.
+
+## [Multichannel communications](./multichannel-communications.md)
+
+A guide to facilitate identification of the correct communication
+channel, both for interacting with the Kubernetes community, and also
+for outbound communication.
+
+## [Social media guidelines](./social-guidelines.md)
+
+Guidelines for social media, currently focused on the
+[K8sContributor](https://twitter.com/k8scontributors) Twitter account
```

Comment:
```suggestion
[@K8sContributors](https://twitter.com/k8scontributors) Twitter account
```

Reply:
Good one, changed in the last commit (this was outdated due to the directory rename).